topdown/crypto.go

```diff
@@ -41,3 +42,67 @@ func builtinCryptoX509ParseCertificates(a ast.Value) (ast.Value, error) {
 func init() {
 	RegisterFunctionalBuiltin1(ast.CryptoX509ParseCertificates.Name, builtinCryptoX509ParseCertificates)
 }
+
+// createRootCAs creates a new Cert Pool from scratch or adds to a copy of System Certs
+func createRootCAs(tlsCaCertFile string, tlsCaCertEnvVar []byte, tlsUseSystemCerts bool) (*x509.CertPool, error) {
```

Comment:
[nit] We could try to be a bit more consistent with the casing of "Ca", and use "CA" everywhere:
```suggestion
func createRootCAs(tlsCACertFile string, tlsCACertEnvVar []byte, tlsUseSystemCerts bool) (*x509.CertPool, error) {
```